kola/tests/rpmostree/deployments.go

```diff
@@ -291,7 +293,7 @@ func rpmOstreeInstallUninstall(c cluster.TestCluster) {
 
 		// check the metadata to make sure everything went well
 		if len(postUninstallStatus.Deployments) != 2 {
-			c.Fatal("Expected two deployments, got %d", len(postUninstallStatus.Deployments))
+			c.Fatal("Expected two deployments, got %d", 2, len(postUninstallStatus.Deployments))
```

Comment:
Missing another `%d` here?

Reply:
Missed removing the `2, `. Had originally changed this to be using the initial length + 1 but that didn't work because of pruning.